provider/dpdk_utils.py

```diff
@@ -0,0 +1,205 @@
+import subprocess
+import paramiko
```

Comment:
Maybe it can be removed? You import it in line 9

Reply:
I wil remove it.